### PR TITLE
17 - What-we-believe

### PR DESCRIPTION
**Pull request by graceschae**
Created belief.js and belief.module.css to create the page. Not sure if the spacing between each of the bolded words and paragraphs is correct, but couldn't figure out how to change it.

### CODE REVIEW

File: cms/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hmccaa/hmcc-web:strapi-0.1.1
+FROM hmccaa/hmcc-web:strapi-0.1.0-arm
```
Comment (thedomcab) [inline cms/Dockerfile:1]: Revert this change. we do want want this to change because different people use different os/cpu platform

File: frontend/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM hmccaa/hmcc-web:gatsby-0.1.1
+FROM hmccaa/hmcc-web:gatsby-0.1.0-arm
```
Comment (thedomcab) [inline frontend/Dockerfile:1]: Revert this change. we do want want this to change because different people use different os/cpu platform

Comment (xumaple) [reply frontend/Dockerfile:1]: Yes, unfortunately I haven't figured out a way to fix this yet :/ let's keep it as just the non-arm one for now...

@graceschae feel free to also put frontend/Dockerfile and cms/Dockerfile into the .gitignore as well so nobody else has to deal with this!

**Review by thedomcab — DISMISSED** [review]
GTG

**Review by xumaple — APPROVED** [review]
